cognite_toolkit/_cdf.py

```diff
@@ -840,6 +845,69 @@ def dump_datamodel_cmd(
     )
 
 
+@feature_flag_app.callback(invoke_without_command=True)
+def feature_flag_main(ctx: typer.Context) -> None:
+    """Commands to enable and disable feature flags for the toolkit."""
+    if ctx.invoked_subcommand is None:
+        print(
+            Panel(
+                "[yellow]Warning: enabling feature flags may have undesired side effects."
+                "\nDo not enable a flag unless you are familiar with what it does.[/]"
+            )
+        )
+        print("Use [bold yellow]cdf-tk feature list[/] or [bold yellow]cdf-tk feature --[flag] --enabled=True|False[/]")
+    return None
+
+
+@feature_flag_app.command("list")
+def feature_flag_list() -> None:
+    """List all available feature flags."""
+
+    cmd = FeatureFlagCommand()
+    cmd.list()
+
+
+@feature_flag_app.command("set")
+def feature_flag_set(
+    flag: Annotated[
+        str,
+        typer.Option(
+            "--flag",
+            "-f",
+            prompt=True,
+            help="Which flag to set",
+        ),
+    ],
+    enabled: Annotated[
+        str,
+        typer.Option(
+            "--enabled",
+            "-e",
+            help="Enable the flag, default is False",
+        ),
+    ],
```

Comment:
Why not a positional argument for the `flag` and the a boolean for `enable` and one for `disable`?
```suggestion
    flag: Annotated[
        str,
        typer.Argument(
            help="Which flag to set",
        ),
    ],
    enable: Annotated[
        bool,
        typer.Option(
            "--enable",
            "-e",
            help="Enable the flag.",
        ),
    ] = False,
    disable: Annotated[
        bool,
        typer.Option(
            "--disable",
            help="Disable the flag.",
        ),
    ] = False,
```

![image](https://github.com/cognitedata/toolkit/assets/60234212/20f7250b-8e11-422c-aa79-80c80f72b924)